src/server/routes/auth.js

```diff
@@ -73,4 +83,45 @@ router.get('/logout', (req, res) => {
 	res.redirect(status.SEE_OTHER, '/');
 });
 
+router.post('/delete-user/:uname', async (req, res) => {
+	// Special Account on MB
+	const USER_DELETER = 'UserDeleter';
+	const USER_DELETER_MBID = 2007538;
+	const {access_token: accessToken} = req.query;
+	const {uname} = req.params;
+	try {
+		const rBody = await request.get('https://musicbrainz.org/oauth2/userinfo').set('Authorization', `Bearer ${accessToken}`).then((rs) => rs.body);
+		if (rBody.metabrainz_user_id !== USER_DELETER_MBID || rBody.sub !== USER_DELETER) {
+			return res.status(401).send();
+		}
+		const {orm} = req.app.locals;
+		const {Editor} = orm;
+		const editor = await Editor.forge({name: uname}).fetch({require: false});
```

Comment:
I had another look at the implementation of the same mechanism in ListenBrainz because I wasn't sure, but it confirmed that we want to delete users by their metabrainz_user_id instead of by name, because the name might be different between website while the meb_user_id will be the same for an account across the websites
```suggestion
		const editor = await Editor.forge({metabrainz_user_id: mbRowId}).fetch({require: false});
```

That'll also require changing the query param name to `mbRowId` as well as modifying the tests accordingly.